code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -9630,3 +9630,7 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	color = "#571212" //like a dark red
 	density = 1.00 //basically water
 	specheatcap = 4.184
+
+/datum/reagent/changeling_blood
```

Comment:
You might want to turn this into a child of the blood reagent, otherwise the changeling will instantly die because it has no blood

Reply:
> You might want to turn this into a child of the blood reagent, otherwise the changeling will instantly die because it has no blood

I'm trying to be crafty and just put some "changeling_blood" into a blood sample for the test and that's it. Currently failing at this because it's obvious before the test.